Integreat/Integreat.Shared/ViewModels/ExtrasContentPageViewModel.cs

```diff
@@ -89,7 +89,8 @@ private async void OnExtraTapped(object obj)
             //special favours for sprungbrett and lehrstellenradar
             if (extra.Alias == "sprungbrett")
                 view = _sprungbrettFactory(extra.Url);
-            else if (extra.Alias == "wohnen" && extra.Post.TryGetValue("api-name", out var apiName) && apiName == "neuburgschrobenhausenwohnraum")
+            else if (extra.Alias == "wohnen" && extra.Post.TryGetValue("api-name", out var apiName)
+                && new [] { "neuburgschrobenhausenwohnraum","testumgebungwohnraum" }.Contains(apiName))

```

Comment:
Das sollten wir für die Zukunft noch überdenken, aber in der Version lassen wir es mal noch enthalten ...